vendor/github.com/docker/notary/README.md

```diff
@@ -98,3 +101,5 @@ $GOPATH/
             docker/
                 notary/
 ```
+
+To build the server and signer, please run `docker-compose build`.
```

Comment:
s/please//

Reply:
vendored code - we can address this in docker/notary